<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AA</class>
 <widget class="QDialog" name="AA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background:url(C:/dhjdf.jpg) ;
}</string>
  </property>
  <widget class="QFrame" name="add_employee">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>441</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background:#51556e;
    border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>D:/QTprojet/smart court.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_id1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>123456</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>NOM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_numero1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="le_prenom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>PRENOM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prenom1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="le_numero">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>NUMERO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nom1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="le_etat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
    font-family:century gothic;
    font-size:24px;
	color:white;
}

</string>
    </property>
    <property name="text">
     <string>ETAT(A ou J)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>342</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:white;
border-radius: 15px;
background:#303030;
}

*:hover
{
	color:#303030;
	border-radius:15px;
	background:white
}</string>
    </property>
    <property name="text">
     <string>ajouter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_etat1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_age1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="le_age">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AGE</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>le_id1</zorder>
   <zorder>le_numero1</zorder>
   <zorder>le_prenom</zorder>
   <zorder>le_prenom1</zorder>
   <zorder>le_numero</zorder>
   <zorder>le_nom1</zorder>
   <zorder>le_etat</zorder>
   <zorder>pb_ajouter</zorder>
   <zorder>le_etat1</zorder>
   <zorder>le_age1</zorder>
   <zorder>le_age</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
